### PR TITLE
docs: update ssr and nextjs

### PR DESCRIPTION
**Pull request by huozhi**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -120,6 +120,10 @@ Turn on/off automatic vendor prefixing (default: `true`)
 - Dynamic styles and themes support
 - CSS Preprocessing via Plugins
 
+## Using in Next.js
+
+Next.js now has SWC compiler and transformer to transile it in fast speed. You don't have to configure it in babel plugins.
```
Comment (styfle) [inline readme.md:125]:
```suggestion
Next.js 12+ includes SWC compiler and transformer. You don't have to configure it in babel plugins.
```

File: readme.md [hunk 2]
```diff
@@ -334,6 +338,13 @@ Please keep in mind that constants defined outside of the component scope are tr
 
 ## Server-Side Rendering
 
+`styled-jsx` v5 introduced `StyledRegistry` component and `useStyleRegistry` hook to let you scope styles rendering in each SSR render to keep concurrent-safe.
+
+- `registry.styles()` will return the array of react components for style tags.
+- `registry.flush()` can clean the existing styles in the registry, it's optional for SSR when you have a standalone registry for each SSR render.
+
+> Next.js 12 has integrated with `styled-jsx` v5 and manage the registry for you.
```
Comment (styfle) [inline readme.md:346]:
```suggestion
> Next.js 12 integrates with `styled-jsx` v5 and manages the registry for you.
```

Comment (huozhi) [reply readme.md]: tense is so hard..

**Review by styfle — APPROVED** [review]

File: readme.md [hunk 3]
```diff
@@ -120,6 +120,10 @@ Turn on/off automatic vendor prefixing (default: `true`)
 - Dynamic styles and themes support
 - CSS Preprocessing via Plugins
 
+## Using in Next.js
+
+Next.js 12+ includes SWC compiler and transformer. You don't have to configure it in babel plugins.
```
Comment (ijjk) [inline readme.md:125]: Even before swc it shouldn't have needed to be configured in babel plugins as Next.js sets it for the user. Might be good to note this. 

```suggestion
Next.js automatically configures `styled-jsx` with babel or swc, you don't have to configure it in babel plugins manually.
```

**Review by ijjk — APPROVED** [review]

**Pull request merged**